Library/Formula/influxdb.rb

```diff
@@ -4,208 +4,126 @@
 class Influxdb < Formula
   desc "Time series, events, and metrics database"
   homepage "http://influxdb.com"
-  url "https://s3.amazonaws.com/get.influxdb.org/influxdb-0.8.8.src.tar.gz"
-  sha1 "a5686d0374ec5ab616e335e9c5fb1aeacd17fb00"
+  url "https://github.com/influxdb/influxdb/archive/v0.9.0.tar.gz"
+  sha256 "c7cc869754d7bfb9374b0a16a9b91073d9588adedbd7bfda1d5709e22d3a3d75"
+  version "0.9.0"
```

Comment:
Don't need this. The URL is simple enough for `brew` to determine a version from.